datadog-integrations-aws-handler/src/datadog_integrations_aws/handlers.py

```diff
@@ -0,0 +1,191 @@
+import logging
+from typing import Any, MutableMapping, Optional
+
+from cloudformation_cli_python_lib import (
+    Action,
+    OperationStatus,
+    ProgressEvent,
+    Resource,
+    SessionProxy,
+)
+from datadog_api_client.v1 import ApiException
+from datadog_api_client.v1.api.aws_integration_api import AWSIntegrationApi
+from datadog_api_client.v1.model.aws_account import AWSAccount
+from datadog_cloudformation_common.api_clients import v1_client
+
+from .models import ResourceHandlerRequest, ResourceModel
+from .version import __version__
+
+# Use this logger to forward log messages to CloudWatch Logs.
+LOG = logging.getLogger(__name__)
+TYPE_NAME = "Datadog::Integrations::AWS"
+TELEMETRY_TYPE_NAME = "integrations-aws"
+
+resource = Resource(TYPE_NAME, ResourceModel)
+test_entrypoint = resource.test_entrypoint
+
+
+def build_aws_account_from_model(model):
+    aws_account = AWSAccount()
+    if model.AccountID is not None:
+        aws_account.account_id = model.AccountID
+    if model.RoleName is not None:
+        aws_account.role_name = model.RoleName
+    if model.AccessKeyID is not None:
+        aws_account.access_key_id = model.AccessKeyID
+    if model.HostTags is not None:
+        aws_account.host_tags = model.HostTags
+    if model.FilterTags is not None:
+        aws_account.filter_tags = model.FilterTags
+    if model.AccountSpecificNamespaceRules is not None:
+        aws_account.account_specific_namespace_rules = model.AccountSpecificNamespaceRules
+    return aws_account
+
+
+@resource.handler(Action.CREATE)
+def create_handler(
+        session: Optional[SessionProxy],
+        request: ResourceHandlerRequest,
+        callback_context: MutableMapping[str, Any],
+) -> ProgressEvent:
+    LOG.info("Starting %s Create Handler", TYPE_NAME)
+    model = request.desiredResourceState
+
+    aws_account = build_aws_account_from_model(model)
+    with v1_client(
+            model.DatadogCredentials.ApiKey,
+            model.DatadogCredentials.ApplicationKey,
+            model.DatadogCredentials.ApiURL or "https://api.datadoghq.com",
```

Comment:
The latest common-python should set the default
```suggestion
            model.DatadogCredentials.ApiURL",
```